maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -141,22 +157,22 @@ private function getRemotePath( $path ) {
 			$this->output( "\tInvalid path to read file content\n" );
 			return null;
 		}
-		
+
 		if ( strpos( $path, 'mwstore' ) === 0 ) {
 			$path = preg_replace( '/mwstore\:\/\/swift-backend\/(.*)\/(images|avatars)/', '', $path );
 		}
-		
+
 		return $path;
 	}
-	
+
 	/**
 	 * Store image in destination path
 	 *
 	 * @param $city_id Int - Wikia ID
 	 * @param $src String - source file
 	 * @param $dst String - destination path
-	 * 
-	 * @return Boolean
+	 *
+	 * @return Boolean|null return false will re-queue the item, null - skip it
```

Comment:
I'd rather write it from the function perspective (not how is later used) like:
Returns false for recoverable error and null for permanent error.